routes/index.js

```diff
@@ -2,8 +2,57 @@ var express = require('express');
 var passport = require('passport');
 var httpProxy = require('http-proxy');
 var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn()
+var crypto = require('crypto');
 var router = express.Router();
 
+
+function formatDate(d) {
```

Comment:
I thought about adding moment.js, but this does the job without an added dependency.

I've tested this code in node.js - toLocaleString has been in JS since version 1.0